src/components/Helmet/Helmet.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+type HelmetProps = {
+  children: React.ReactNode;
+  title: string;
+};
+
+const Helmet: React.FC<HelmetProps> = (props) => {
```

Comment:
I don't quite understand purpose of this component. All it does is create a div with class w-108 wrapper around any content passed to it.

There's a react-helmet library that provides a Helmet component. https://www.npmjs.com/package/react-helmet
That component manages all of your changes to the document head. It's handy for handling page titles, meta tags, and anything else you might want to add to the head of your document. It's especially useful for dealing with SEO (Search Engine Optimization) in React applications, where you want to ensure that the metadata is correctly set for each page.